tests/DoctrineORMModuleTest/Yuml/YumlControllerFactoryTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace DoctrineORMModuleTest\Yuml;
+
+use DoctrineORMModule\Yuml\YumlController;
+use DoctrineORMModule\Yuml\YumlControllerFactory;
+use Zend\ServiceManager\AbstractPluginManager;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+class YumlControllerFactoryTest extends \PHPUnit_Framework_TestCase
+{
+    public function testCreateService()
+    {
+        $config = [
+            'zenddevelopertools' => [
+                'toolbar' => [
+                    'enabled' => true
+                ]
+            ]
+        ];
+
+        $serviceLocator = $this->getMockBuilder(ServiceLocatorInterface::class)
+            ->getMock();
+        $pluginManager = $this->getMockBuilder(AbstractPluginManager::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $serviceLocator->expects(static::any())->method('get')->with('config')->willReturn($config);
```

Comment:
IMHO it's not important for the test scope, it would be just a constraint.  
But probably I should integrate more tests on this factory. I'm going to write those tests.